### DISCUSSION TITLE
Get all products by location

### DISCUSSION THREAD

**Discussion by snappy316** [weather-gov/api#208] [Q&A]
:wave: I'm wondering if it's possible to get all products for a given location? I'd like to be able to hit a `/products/locations/{locationId}` endpoint (which currently returns a 404). This seems like the only missing "products / types / locations" permutation! I'd imagine it would be similar to the existing `/products/types/{typeId}` endpoint?

I'm basically looking for an API-based way to mirror the functionality the WFO sites use for their "Latest Text Products Issued by {WFO}" dropdown lists (e.g., [Seattle](https://www.weather.gov/sew/), [Las Vegas](https://www.weather.gov/vef/)). The only alternative I can think of is to hit `/products/locations/SEW/types`, then loop through the returned type IDs to hit `/products/types/{typeId}/locations/SEW` to pull each type ID's latest product info...

**Comment by JJKraw** [comment]
You can use /products?location={WFO}. It returns a lot of data by default, so you may want to include time and/or limit parameters also. You also get products that normally aren't useful to us mere mortals (like HADS Data), so there might be some filtering of the data you receive.

**Comment by snappy316** [comment]
Thanks so much for the tip @JJKraw - I can work with that!!
